src/routes/(app)/checkout/+page.svelte

```diff
@@ -91,6 +91,7 @@
 				input.reportValidity();
 
 				event.preventDefault();
+				submitting = false;
```

Comment:
put the whole `for` inside a `try` block, and add a `finally` block afterwards with `submitting = false`